### DISCUSSION TITLE
Nextcloud on Windows 11- Error in launching

### DISCUSSION THREAD

**Discussion by debatoshk** [nextcloud/all-in-one#3996] [Questions] [answered]
Hi All,

Wishing you a very healthy and happy New year. 

I am a newbie into the Nextcloud and Homelab environment.  Was trying to set up  a Nextcloud server using my old Dell Optiplex 3060 Core i3 mini pc. Had read many articles on setting up the Nextcloud server on Windows 11. Have a domain and configured a Cloudflare tunnel. Did ran the command to install the connector and that seemed worked as it indicated as "Healthy" on the Cloudflare dashboard. The intention is to install the Nextcloud and access it worldwide.

During the installation of NextCloud AIO, each time was getting stuck on the domain page and was unable to proceed further. However, saw an article and found that the domain validation can be skipped. Also wanted to install in Drive "D" and used the below code. 

docker run ^
--sig-proxy=false ^
--name nextcloud-aio-mastercontainer ^
--restart always ^
--publish 80:80 ^
--publish 8080:8080 ^
--publish 8443:8443 ^
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config ^
--volume //var/run/docker.sock:/var/run/docker.sock:ro ^
-e NEXTCLOUD_DATADIR="/run/desktop/mnt/host/d/ncdata" ^
--env SKIP_DOMAIN_VALIDATION=true ^
nextcloud/all-in-one:latest

This enabled to skip the domain validation and I had proceeded with the installation and had applied the domain name as obtained from the Cloudflare account.
![1](https://github.com/nextcloud/all-in-one/assets/155521585/dd7e13cd-d9f6-4ca9-a9b8-ad1cbf6af97a)

![2](https://github.com/nextcloud/all-in-one/assets/155521585/11db7d8d-d786-4ca3-a65a-3cb56c6b26bc)

However, when attempting to log in, the below error message is appearing.
![3](https://github.com/nextcloud/all-in-one/assets/155521585/c43a657a-c3bb-4a8b-b5b6-6ea2370ac1b5)

The settings related to the Cloudflare Tunnel and the lists of the containers which are running.
![4](https://github.com/nextcloud/all-in-one/assets/155521585/e3478303-0227-4500-b6b1-bf9f32a4de86)

![5](https://github.com/nextcloud/all-in-one/assets/155521585/27793fe4-7704-4e75-b010-67194a04c404)

Also the port for 443 and 3478 are forwarded.
![6](https://github.com/nextcloud/all-in-one/assets/155521585/045909ae-4897-447d-86f2-9334ce72d505)

If anyone could help me on this, it would be a great help.

Thank you in advance.

Cheers.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-run-nextcloud-behind-a-cloudflare-tunnel
